perun-core/src/main/java/cz/metacentrum/perun/core/api/AuditMessagesManager.java

```diff
@@ -3,58 +3,53 @@
 import java.util.List;
 import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
 import cz.metacentrum.perun.core.api.exceptions.PrivilegeException;
-import cz.metacentrum.perun.core.api.exceptions.WrongRangeOfCountException;
 import java.util.Map;
 
-
 /**
- * AuditMessagesManager manages audit messages (logs).
+ * AuditMessagesManager manages audit messages (logs). Entry Logic interface.
  *
  * @author Michal Stava
  */
 public interface AuditMessagesManager {
 
-	public final static Integer COUNTOFMESSAGES=100;
+	Integer COUNTOFMESSAGES = 100;
```

Comment:
As far as I know, constants should be annotated ``public/private static final``, please do not remove it.